include/mbedtls/cmac.h

```diff
@@ -141,16 +156,22 @@ int mbedtls_cipher_cmac( const mbedtls_cipher_info_t *cipher_info,
 
 #if defined(MBEDTLS_AES_C)
 /**
- * \brief           AES-CMAC-128-PRF
- *                  Implementation of (AES-CMAC-PRF-128), as defined in RFC 4615
- *
- * \param key       PRF key
- * \param key_len   PRF key length in bytes
- * \param input     buffer holding the input data
- * \param in_len    length of the input data in bytes
- * \param output    buffer holding the generated pseudorandom output (16 bytes)
- *
- * \return          0 if successful
+ * \brief           This function generates pseudorandom output.
```

Comment:
Blocker: The first line of the function description is too generic, I'd prefer the identifier `AES-CMAC-PRF-128` be mentioned earlier. What about `This function implements the AES-CMAC-PRF-128 pseudorandom function`.

Reply:
Fixed.